smithy-cli/src/main/java/software/amazon/smithy/cli/commands/DiffCommand.java

```diff
@@ -51,7 +52,7 @@ public String getName() {
 
     @Override
     public String getSummary() {
-        return "Compares two Smithy models and reports differences.";
+        return "Compares two Smithy models and detects differences.";
```

Comment:
The previous wording reads more accurate to me, as detecting does not imply reporting.

Reply:
Changed it back to reports